x509-identity-mgmt/ejbca/bin/before-shutdown.sh

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash
 
+##################################################################
```

Comment:
I think we could add a NOTICE file saying that some files were developed by PrimeKey and changed by us(list these files) and they are licensed under GPL v2.1 in order to avoid some misunderstandings.

Reply:
Done!